src/components/training/TrainingPlanOverview.jsx

```
@@ -0,0 +1,1114 @@
+                                  <input
+                                    type="text"
+                                    value={editingDay.name}
+                                    onChange={(e) =>
+                                      setEditingDay({
+                                        ...editingDay,
+                                        name: e.target.value,
+                                      })
+                                    }
+                                    className="w-full bg-zinc-800/50 border border-zinc-600 rounded-lg px-4 py-2 text-white text-2xl font-bold backdrop-blur focus:border-blue-500 focus:outline-none"
+                                    placeholder="Day name"
+                                    autoFocus
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)